src/screens/AddCryptoScreen/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react';
+import {Text} from 'react-native';
+import {
+  Button,
+  Input,
+  ContainerScreen,
+  ButtonAdd,
+  AddText,
+  Title,
+} from './styles';
+import theme from '../../utils/themes';
+import {useDispatch, useSelector} from 'react-redux';
+import {addNewCrypto} from '../../store/actions';
+
+const AddNewCrypto = ({navigation}) => {
+  const cryptos = useSelector(state => state.cryptos);
+  const [input, setInput] = useState<string>('');
```

Comment:
unnecessary type if you already pass a basic value it will take the type of the value (string, boolean, number, [], etc)